packages/meditrak-app/app/assessment/selectors.js

```diff
@@ -109,3 +118,80 @@ export const getQuestion = (state, questionId) => state.assessment.questions[que
 export const getAnswers = state => state.assessment.answers;
 
 export const getAnswerForQuestion = (state, questionId) => getAnswers(state)[questionId];
+
+const getArithmeticResult = (state, config) => {
+  const { formula, valueTranslation, defaultValues = {}, text = '' } = config.calculated;
+  const values = {};
+  const variables = getVariables(formula);
+  let translatedText = text;
+
+  variables.forEach(questionIdVariable => {
+    let answer = getAnswerForQuestion(state, questionIdVariable) || 0;
```

Comment:
```suggestion
    let answer = getAnswerForQuestion(state, questionIdVariable);
```
You've got logic below that relies on `undefined` being left as is